### PR TITLE
Update basic-kotlin-native-app-codeblocks-code.md

### PR DESCRIPTION
**Pull request by Mazorius**
* Update kotlin multiplatform to 1.3.72
* change mavenCentral() to jcenter()
* Update Gradle to 6.7

### CODE REVIEW

File: pages-includes/docs/tutorials/native/basic-kotlin-native-app-codeblocks-code.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 
 ```
 plugins {
-    id 'org.jetbrains.kotlin.multiplatform' version '1.3.21'
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.72'
```
Comment (p7nov) [inline pages-includes/docs/tutorials/native/basic-kotlin-native-app-codeblocks-code.md:7]: Let's use the placeholder for the latest stable version `{{ site.data.releases.latest.version }`:
```suggestion
    id 'org.jetbrains.kotlin.multiplatform' version '{{ site.data.releases.latest.version }'
```

File: pages-includes/docs/tutorials/native/basic-kotlin-native-app-codeblocks-code.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ kotlin {
 }
 
 wrapper {
-  gradleVersion = "5.3.1"
+  gradleVersion = "6.7"
```
Comment (p7nov) [inline pages-includes/docs/tutorials/native/basic-kotlin-native-app-codeblocks-code.md:23]: There is a placeholder for Gradle version as well -  `{{ site.data.releases.tutorials.native.gradle_version }}`
However, it's now outdated. Please change the version here and update the placeholder in [/data/releases.yml](https://github.com/JetBrains/kotlin-web-site/blob/bd50bea578ed517765d18876d52accc93bd8d237/data/releases.yml#L25)

**Review by p7nov — CHANGES_REQUESTED** [review]
Hi! 
Thanks for the update!
I suggest making these samples even better by using variables for version numbers. They will update automatically and we won't run into this again.
Please see my comments and apply them to other occurrences of the same lines.

**Comment by Mazorius** [comment]
Hi @p7nov 

I add the requested changes. I also change Gradle to the `BIN` distribution as `ALL` seems a bit to much.

Hopefully this suits your request ;-)

**Review by p7nov — APPROVED** [review]

**Comment by p7nov** [comment]
Thank you for this update!

**Pull request merged**